Added lates stable node and configuration txt (#796) (#796)
nuclia/frontend: README.md

Start by creating an account with an email and password (as SSO doesn't work locally).

> In `apps/dashboard/src/environments_config`, create a file `local-stage/app-config.json` with the correct configuration. <sub> You will have to speak to a supervirsor so they can give the proper cofiguration</sub>

Reviewer: ```suggestion
In `apps/dashboard/src/environments_config`, create a file `local-stage/app-config.json` with the correct configuration. <sub>Ask a supervisor to get a proper configuration.</sub>
```
